rai_test_utils/rai_test_utils/utilities/utils.py

```diff
@@ -29,7 +32,7 @@ def retrieve_dataset(dataset, **kwargs):
     :rtype: object
     """
     # if data not extracted, download zip and extract
-    outdirname = 'datasets.4.27.2021'
+    outdirname = DOWNLOADED_DATASET_DIR
     if not os.path.exists(outdirname):
```

Comment:
if the dataset is downloaded we don't re-download it here... we just re-use the downloaded one